src/language/walker/ruleWalker.ts

```diff
@@ -82,6 +82,14 @@ export class RuleWalker extends SyntaxWalker {
         }
     }
 
+    public createFix(replacements: Replacement[]): Fix {
```

Comment:
I'd prefer to expose only the typical fix methods. These are what we need for the current check and expose less API details to all the rules that might start using it.

so far we need
`appendText(n: node, t: text)` for putting a semicolon after the statement, and
`delete(start: number, length: number)` for deleting a semicolon?